packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Updated `ProfileInfo.updateProperty` function to support updating properties in typeless profiles. [#2196](https://github.com/zowe/zowe-cli/issues/2196)
```

Comment:
Since this PR will become another prerelease, I'm curious if we should want to just link to the other version number (since it's just a port) 😋 

Similar to what we did for ```## `8.0.0-next.202407021516` ``` > ``` `- Update: See `5.25.0` for details` ```